src/components/ContentViewer.tsx

```
@@ -0,0 +1,111 @@
+import { Textarea } from "../components/ui/textarea";
+import MarkdownRenderer from "../utils/markdownRenderer";
+
+// Replace showEdited with "mode"
+interface ContentViewerProps {
+  mode: "original" | "revised";
+  pdfUrl: string;
+  revisedText: string;
+  isEditing: boolean;
+  setRevisedText: (text: string) => void;
+  loading: boolean;
+  pagesContent: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)